ui/src/dashboards/components/TemplateVariableRow.js

```diff
@@ -30,7 +32,9 @@ const RowValues = ({
   onStartEdit,
   autoFocusTarget,
 }) => {
-  const _values = values.map(({value}) => value).join(', ')
+  const _values = uniq(values.map(({value}) => value))
```

Comment:
Yeah, I think you'd have to convert to a set, then back to an array. I'm happy with `uniq`